platforms/nodejs/Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM buildpack-deps:stretch
+ARG DEBIAN_FLAVOR
+FROM buildpack-deps:${DEBIAN_FLAVOR}
+ARG DEBIAN_FLAVOR
```

Comment:
duplicate arguments?

Reply:
no .. we would need one after the from as well ... first arg is to be used in the from statement second one will be used for env variable